transit_odp/browse/templates/browse/timetables/dataset_detail/review_line_metadata.html

```diff
@@ -115,31 +115,12 @@ <h1 class="govuk-heading-xl app-!-mb-4 dont-break-out">
     {% if is_timetable_visualiser_active %}
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-full">
-                <h2 class="govuk-heading-m">Timetable</h2>
-                <h2 class="govuk-heading-m">Filter by</h2>
-                <div class="govuk-form-group  govuk-!-width-one-quarter govuk-!-padding-0  ">
-                <label class="govuk-label" for="timetable_date">
-                    Date
-                </label>
-                <span id="timetable_date-hint" class="govuk-hint ">
-                    For example: 21/11/2014
-                </span>            
-                <input type="text"            
-                id="timetable_date" name="start" style="width: 90%;" class="govuk-input" aria-describedby="timetable_date-hint"
-                value={{curr_date}} data-gtm-form-interact-field-id="0" />
-                <img id="timetable_date_calendar"
-                class="govuk-!-margin-right-1" style="position: relative;top: 2px;margin-left: -30px;" 
-                width="20" height="20" src="{% static 'frontend/images/calendar-icon.png' %}"
-            alt="{% trans 'Calendar icon' %}">
-            <button type="submit" class="govuk-button govuk-!-margin-top-2" data-module="govuk-button" onclick="BODSFrontend.changeTargetDate('timetable_date');">
-                Apply filter
-            </button>
-            </div>
+                {% include 'publish/dataset_detail/timetable_filters.html' %}
             {% if not outbound_timetable.empty %}
-                {% include 'publish/dataset_detail/journey_stop_timetable.html' with df=outbound_timetable journey_name=outbound_journey_name show_all=outbound_show_all total_rows=outbound_total_row_count total_page=outbound_total_page curr_page=outbound_curr_page page_param='outboundPage' show_all_param='showAllOutbound'  %}
+                {% include 'publish/dataset_detail/timetable_journey_stop.html' with direction='outbound' df=outbound_timetable journey_name=outbound_journey_name stops=outbound_stops show_all=outbound_show_all total_rows=outbound_total_row_count total_page=outbound_total_page curr_page=outbound_curr_page page_param='outboundPage' show_all_param='showAllOutbound'  %}
```

Comment:
there are a few parameters passed to the page - can we move some of them to a dict and pass it to the page so that its more readable?

Reply:
ok ... let me make it generic to pass the params as all dict, in order to iterate over the same.